instrumentation/active_record/lib/opentelemetry/instrumentation/active_record/instrumentation.rb

```diff
@@ -22,11 +21,7 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         end
 
         compatible do
-          # We know that releases after MAX_MAJOR_VERSION are unstable so we
-          # check the major version number of the gem installed to make sure we
-          # do not install on a pre-release or full release of the latest
-          # if it exceeds the MAX_MAJOR_VERSION version.
-          gem_version >= MINIMUM_VERSION && gem_version.segments[0] <= MAX_MAJOR_VERSION
```

Comment:
Why did you decide to remove the MAX_MAJOR_VERSION logic?

Reply:
It's the only place we do it and it felt asymmetric